rpcmiddleware/proto.go

```diff
@@ -54,7 +54,7 @@ func RPCErr(req *lnrpc.RPCMiddlewareRequest,
 	err error) (*lnrpc.RPCMiddlewareResponse, error) {
 
 	if err != nil {
-		return RPCErrString(req, err.Error())
+		return RPCErrString(req, err.Error(), nil)
```

Comment:
I think you should add these changes which are intended to solve the linter issue in a separate commit prior to the `golang` bump commit, and add a description to the commit to explain why they are needed.